public/assets/transition-page/styles/transition-page.css

```diff
@@ -71,6 +71,7 @@ button {
 
 #main-container {
   margin-top: min(106px, 12vh);
+  margin-bottom: min(106px, calc(12vh + 61px));
```

Comment:
not sure if such complex calculations are entirely necessary.

Reply:
Okay, I am assuming that the complex part is `calc(12vh + 61px`. The `61px` (which is the fixed height for the bottom banner) is used to make sure at super short device height, the bottom banner does not cover the main content on the page (as the bottom banner is fixed at the bottom using `position: fixed`, and height is "floating"). That is the best I could think of so far, and will improve it if something better comes to mind!